examples/tv-app/android/App/platform-app/src/main/java/com/matter/tv/server/MatterCommissioningPrompter.java

```diff
@@ -10,8 +10,8 @@
 import android.widget.EditText;
 import androidx.appcompat.app.AlertDialog;
 import androidx.core.app.NotificationCompat;
-import com.tcl.chip.tvapp.UserPrompter;
-import com.tcl.chip.tvapp.UserPrompterResolver;
+import com.matter.tv.server.tvapp.UserPrompter;
```

Comment:
since this is an example app, should this be `com.examples.matter.tv.server` or similar?

Reply:
ok. I guess better matter than tcl.